From c49aea79928f4761b9cd6bceefc2232ec3d0ba88 Mon Sep 17 00:00:00 2001
From: dirk <dirk@git.imagemagick.org>
Date: Thu, 12 May 2016 21:55:05 +0200
Subject: [PATCH] Raise warning instead of error because we return the image(s)
 read so far.

(cherry picked from commit c023c438c61801e68ea86efc93c3577269f1a1ac)

This is needed for fixing CVE-2016-5687

origin: upstream,https://github.com/ImageMagick/ImageMagick/commit/c023c438c61801e68ea86efc93c3577269f1a1ac
bug-debian: https://bugs.debian.org/832890
---
 coders/dds.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: imagemagick-6.7.7.10/coders/dds.c
===================================================================
--- imagemagick-6.7.7.10.orig/coders/dds.c	2016-11-14 09:58:04.721136163 -0500
+++ imagemagick-6.7.7.10/coders/dds.c	2016-11-14 09:58:04.717136163 -0500
@@ -896,7 +896,7 @@
   */
   if (EOFBlob(image) != MagickFalse)
     {
-      ThrowFileException(exception,CorruptImageError,"UnexpectedEndOfFile",
+      ThrowFileException(exception,CorruptImageWarning,"UnexpectedEndOfFile",
         image->filename);
       return(MagickFalse);
     }
